src/openloco/windows/promptbrowsewnd.cpp

```diff
@@ -188,11 +188,11 @@ namespace openloco::ui::prompt_browse
 
         textinput::cancel();
 
-        _type = (uint8_t)type;
-        _fileType = (uint8_t)browse_file_type::saved_game;
+        *_type = type;
```

Comment:
`_type` can be integrated

Reply:
The same probably goes for a bunch more of the locals in this file. I'd like to leave that for another PR.